content/docs/en/elements/components/date-picker.md

```diff
@@ -1,5 +1,13 @@
 ---
 title: DatePicker
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_date_picker_.datepicker
 ---
 
-TODO
+The DatePicker allows the user to pick a date.
+
+---
+
+```html
+<DatePicker />
+```
+[> screenshots for=DatePicker <]
```

Comment:
Should have a table of properties according to https://docs.nativescript.org/api-reference/modules/_ui_date_picker_

- `date`
- `minDate`
- `maxDate`
- `day`
- `month`
- `year`

Reply:
This would be one of those components that I have personally never implemented before - so I think I'll update things like this once I actually thoroughly run through them ;-)